tests/macros/test-EmbedLiveSample.js

```diff
@@ -59,6 +59,18 @@ describeMacro('EmbedLiveSample', function () {
             '</iframe>'
         );
     });
+    itMacro('One argument: XSS attempt (failed)', function (macro) {
+        macro.ctx.env.live_samples = {'base_url': 'https://mdn.mozillademos.org'};
```

Comment:
Now that you have this in the `beforeEachMacro` call above, it can be removed.

Reply:
🤦‍♂️ of course, good suggestion.